pkg/infra/usagestats/usage_stats.go

```diff
@@ -290,6 +290,15 @@ func (uss *UsageStatsService) updateTotalStats() {
 	}
 }
 
+func (uss *UsageStatsService) shouldBeReported(dsType string) bool {
+	ds, ok := plugins.DataSources[dsType]
```

Comment:
Since now we rely on global state, tests are failing. So...

1) I can update the expected values to make tests pass temporally (all of them are considered as "others", because no test is registered / signed in the context of that test).
2) Create a new service / interface, that internally will access that global state, but it'd be potentially mockable / injectable to the `UsageStatsService` to be able to write some unit tests. But that'd exist only for testing purposes, and probably it'd be needed to be changed as soon as we deal with the global state.
3) Maybe I can somehow override that global state for this specific test (not sure how, neither what's recommended here -safe, etc-). For example, setting some values and then cleaning them with `t.Cleanup`, but since tests from different packages are executed concurrently, it may lead to some other tests to fail.

Thoughts?? Thanks!! 🙏🏻